rllib/algorithms/ppo/ppo.py

```diff
@@ -464,10 +464,11 @@ def _training_step_new_api_stack(self) -> ResultDict:
                     from_worker_or_learner_group=self.learner_group,
                     policies=policies_to_update,
                     global_vars=None,
+                    inference_only=True,
```

Comment:
Yeah, I think "inference_only" is the best term. Most descriptive and still short.

Also, I like that the "learner" version is the default. Matches the use-experience for a typical RL setup (you normally train your model in one or another way first, before deploying it to prod).